Toolbar/Add Stamps to Menu/Sample_Project/packages.config

```diff
@@ -0,0 +1,8 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
<img width="262" alt="image" src="https://user-images.githubusercontent.com/65169077/153593682-dbabbada-45c7-43a8-9ad0-dac03a4541c3.png">

Name the project name meaningfully and also keep the sln and .csprojec in the same folder

Reply:
Changed the name and moved both in the same folder sir.